require/resolve.go

```diff
@@ -13,24 +12,24 @@ import (
 
 const NodePrefix = "node:"
 
+func (r *RequireModule) resolvePath(base, name string) string {
+	if r.r.pathResolver != nil {
```

Comment:
you could change the constructor to default to the `DefaultPathResolver` instead of checking this every time. e.g.
```
func NewRegistry(opts ...Option) *Registry {
	r := &Registry{
           pathResolver : DefaultPathResolver
        }
        ...

Reply:
This would break the use of zero-value `Registry`. A similar mechanism is already implemented for SourceLoader.